charts/trusted-issuers-list/templates/post-hook-initdata.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.initData.initEnabled -}}
```

Comment:
In theory we could check here {{ - if .Values.initData.issuers  -}} to only generate this when issuers are configured. Then we could remove the enabled flag. But on the other side not sure if having flag is cleaner

Reply:
I think it's cleaner, and did it the same way in orion. I would keep it this way.